python/python/glide/async_commands/sorted_set.py

```diff
@@ -169,6 +170,145 @@ def __init__(
         self.limit = limit
 
 
+class GeospatialData:
+    """
+    Represents a geographic position defined by longitude and latitude.
+
+    The exact limits, as specified by EPSG:900913 / EPSG:3785 / OSGEO:41001 are the following:
+        - Valid longitudes are from -180 to 180 degrees.
+        - Valid latitudes are from -85.05112878 to 85.05112878 degrees.
+
+    Args:
+        longitude (float): The longitude coordinate.
+        latitude (float): The latitude coordinate.
+    """
+
+    def __init__(self, longitude: float, latitude: float):
+        self.longitude = longitude
+        self.latitude = latitude
+
+
+class GeoUnit(Enum):
+    """
+    Enumeration representing distance units options for the `GEODIST` command.
+    """
+
+    METERS = "m"
+    """
+    Represents distance in meters.
+    """
+    KILOMETERS = "km"
+    """
+    Represents distance in kilometers.
+    """
+    MILES = "mi"
+    """
+    Represents distance in miles.
+    """
+    FEET = "ft"
+    """
+    Represents distance in feet.
+    """
+
+
+class OrderBy(Enum):
```

Comment:
Please align with #1439 - this enum is probably moved or file is renamed. You can keep it as is for now, but don't forget to update it later to avoid having 2 `OrderBy(Enum)` in the client.